_VERSION_MIGRATION/migrate-to-next.py

```diff
@@ -98,11 +99,13 @@ def get_containers(containers):
 
 def get_spec(config):
     containers = [config]
+    terminationGracePeriod = "100s"
     if config.get('containers'):
         containers = config['containers']
-
+    if "terminationGracePeriod" in config:
```

Comment:
In maestro v9 the name of the field is `shutdownTimeout`

Reply:
Oh, good to know. Thank you!